src/options/parallel_options.toml

```diff
@@ -0,0 +1,62 @@
+id     = "PARALLEL"
+name   = "Parallel"
+
+[[option]]
+  name       = "computePartitions"
+  category   = "expert"
+  long       = "compute-partitions=N"
+  type       = "uint64_t"
+  default    = "0"
+  help       = "make n partitions. n <2 disables computing partitions entirely"
+
+[[option]]
+  name       = "checksBeforePartitioning"
+  category   = "expert"
+  long       = "checks-before-partition=N"
+  type       = "uint64_t"
+  default    = "1"
+  help       = "number of standard or full effort checks until partitioning"
+
+[[option]]
+  name       = "partitionConflictSize"
+  category   = "expert"
+  long       = "conflict-size=N"
+  type       = "uint64_t"
+  default    = "0"
+  help       = "number of literals in a cube"
+
+[[option]]
+  name       = "partitionStrategy"
+  alias      = ["partition"]
+  category   = "expert"
+  long       = "partition-strategy=MODE"
+  type       = "PartitionMode"
+  default    = "REVISED"
+  help       = "choose partition strategy mode"
+  help_mode  = "Partition strategy modes."
+[[option.mode.REVISED]]
+  name = "revised"
+  help = "For 4 partitions, creates cubes C1, C2, C3, !C1 & !C2 & !C3"
+
+[[option]]
+  name       = "partitionCheck"
+  alias      = ["check"]
+  category   = "expert"
+  long       = "partition-check=MODE"
+  type       = "CheckMode"
+  default    = "STANDARD"
+  help       = "select whether partitioning happens at full or standard check"
+  help_mode  = "Partition check modes."
+[[option.mode.STANDARD]]
+  name = "standard"
+  help = "create partitions at standard checks"
+[[option.mode.FULL]]
+  name = "full"
+  help = "create partitions at full checks"
+
+[[option]]
+  name       = "writePartitionsToFileName"
+  category   = "expert"
+  long       = "write-partitions-to=N"
+  type       = "std::string"
```

Comment:
I just noticed that this is doing essentially the same as `--out`. It might make sense to use the `ManagedOut` utility here as well: it would take care of managing the file stream, or using `std::out`. You could then simply say `*options().parallel.partitionsOut << "foo" << std::endl;`. (note the `*`)

See https://github.com/cvc5/cvc5/blob/master/src/options/base_options.toml#L22 for how to specify the option.